packages/web/src/scss/components/Tag/_Tag.scss

```diff
@@ -4,10 +4,7 @@
 @use 'tools';
 
 .Tag {
-    @include typography.generate(map.get(theme.$sizes, medium, typography));
```

Comment:
This is OK and shouldn't be removed. We usually don't rely on inheriting the typography in components.